packages/merge-queue/README.md

```diff
@@ -0,0 +1,28 @@
+# merge-queue
+
+This is a small bot for managing merge queue for repositories.
```

Comment:
Please mention that the moment [Github's merge queue](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue) becomes available to us, we'll delete this bot.

Reply:
There is a possibility that we cannot just use Github's merge queue as a replacement. But yeah if Github provides what we want, we can delete this bot.

Reply:
Added a sentence